### PR TITLE
6489 migrate all medic docs references 

### PR DESCRIPTION
**Pull request by ronna**
# Description

Replace references to medic-docs and  link to the appropriate location in the CHT docs site

medic/cht-core#6489

# Code review items

- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: config/default/GUIDE.md [hunk 1]
```diff
@@ -100,4 +100,4 @@ To add a condition card, add it to the "cards" section.
 To add a field inside an existing condition card, add it to the "fields" section of that card.
 You can remove the fields or condition cards that you don't want to show by deleting the whole code block belonging to that field/card.
 
-For more information: [Condition Cards documentation](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md#cards)
+For more information: [Condition Cards documentation](http://localhost:1313/apps/features/contacts/#condition-cards)
```
Comment (abbyad) [inline config/default/GUIDE.md:103]: Looks like localhost should be replaced by `https://docs.communityhealthtoolkit.org`

Comment (ronna) [reply config/default/GUIDE.md:103]: Hi @abbyad I have made that change and other changes in this commit: https://github.com/ronna/cht-core/commit/e7cbc33769ca04413ee7470af42bf05608e679bd how do I make another pull request

**Comment by ronna** [comment]
The below were not updated, mainly links to pdf files which I could not locate on docs.communityhealthtoolkit.org

https://github.com/medic/cht-core/blob/87d6db9c51e1b2f12d5b34e20497f94a165eb12c/release-notes/docs/3.8.0.md
line 29

https://github.com/medic/cht-core/blob/8322907698ab8ccbb9b930330dc074ab268e45cb/release-notes/docs/3.2.0.md
line 18

https://github.com/medic/cht-core/blob/dfeee0069d8f026af249b42ce7fabbdc1bfcd8b8/release-notes/docs/3.4.0.md
line 44

**Comment by ronna** [comment]
The below files are missing in my forked branch, how do I update my branch so as to get the files:
https://github.com/medic/cht-core/blob/f72dc013f7c4d5ee781f7bea03f782401890cca1/release-notes/docs/3.8.2.md
https://github.com/medic/cht-core/blob/1c40bd6e850ab373b500108055a60c33a1361894/release-notes/docs/3.9.1.md

**Comment by ronna** [comment]
https://github.com/medic/cht-core/blob/12981f5b6f811eae155d8d5005f7c50c86842aa4/admin/src/js/controllers/export-dhis.js

I am not sure which is the right URL for DHIS2 Integration

https://docs.communityhealthtoolkit.org/apps/features/integrations/dhis2/ or https://docs.communityhealthtoolkit.org/apps/guides/integrations/dhis2-aggregate/

File: config/default/GUIDE.md (hunk 1 above)
Comment (abbyad) [reply config/default/GUIDE.md:103]: As long as you make the commits in the same branch you don't need to make a further pull request. Seems like that's what you are doing, so it's all good!

**Comment by abbyad** [comment]
1. For the PDFs, the content seems as though it should be in the new CHT docs. If you can't find it, it would be good to create a [documentation issue in cht-docs](https://github.com/medic/cht-docs/issues/new).
1. To add the missing files you'll need to rebase your branch. Are you able to do that?
1. For [DHIS](https://github.com/medic/cht-core/blob/12981f5b6f811eae155d8d5005f7c50c86842aa4/admin/src/js/controllers/export-dhis.js) I would link to this one: https://docs.communityhealthtoolkit.org/apps/features/integrations/dhis2/

**Comment by ronna** [comment]
Hi @abbyad 
> 1. For the PDFs, the content seems as though it should be in the new CHT docs. If you can't find it, it would be good to create a [documentation issue in cht-docs](https://github.com/medic/cht-docs/issues/new).

done, issue created: https://github.com/medic/cht-docs/issues/323

> 2. To add the missing files you'll need to rebase your branch. Are you able to do that?
Yes, and I have updated the URL

> 3. For [DHIS](https://github.com/medic/cht-core/blob/12981f5b6f811eae155d8d5005f7c50c86842aa4/admin/src/js/controllers/export-dhis.js) I would link to this one: https://docs.communityhealthtoolkit.org/apps/features/integrations/dhis2/

URL updated to the correct one

File: config/default/GUIDE.md (hunk 1 above)
Comment (abbyad) [reply config/default/GUIDE.md:103]: I still don't this updated to the docs website. I'll fix it here with a suggestion:

```suggestion
For more information: [Condition Cards documentation](http://docs.communityhealthtoolkit.org/apps/features/contacts/#condition-cards)
```

**Comment by abbyad** [comment]
Looking almost ready to merge! I noticed that there are a lot of moved files:
image

Do you know what the story is with those?

**Comment by ronna** [comment]
Hi @abbyad 
> Do you know what the story is with those?

Sorry I have no idea, maybe in the changes were made before I rebased the code on Master and after rebase showed up.

**Comment by abbyad** [comment]
Ok, thanks Ronald! Are all the changes you wanted to make complete? If so, are you able to resolve the conflicts so that we can merge your PR?

### Conflicting files
```
.github/PULL_REQUEST_TEMPLATE.md
sentinel/server.js
sentinel/src/db.js
sentinel/src/lib/feed.js
sentinel/src/lib/metadata.js
sentinel/src/schedule/index.js
sentinel/src/schedule/outbound.js
sentinel/src/schedule/replications.js
sentinel/tests/unit/lib/feed.js
sentinel/tests/unit/lib/metadata.js
sentinel/tests/unit/schedule/outbound.js
sentinel/tests/unit/schedule/replications.js
```

**Comment by ronna** [comment]
> Ok, thanks Ronald! Are all the changes you wanted to make complete?

Yes all the changes are complete

 If so, are you able to resolve the conflicts so that we can merge your PR?

my attempts to resolve the conflicts were fruitless, your guidance will be greatly appreciated:

$ git pull --rebase upstream master
From https://github.com/medic/cht-core
 * branch            master     -> FETCH_HEAD
First, rewinding head to replay your work on top of it...
Applying: Migrate all medic-docs references #6489
Using index info to reconstruct a base tree...
M       .github/PULL_REQUEST_TEMPLATE.md
M       sentinel/server.js
M       sentinel/src/db.js
M       sentinel/src/lib/feed.js
M       sentinel/src/lib/metadata.js
M       sentinel/src/schedule/index.js
M       sentinel/src/schedule/outbound.js
M       sentinel/src/schedule/replications.js
M       sentinel/tests/unit/lib/feed.js
M       sentinel/tests/unit/lib/metadata.js
M       sentinel/tests/unit/schedule/outbound.js
M       sentinel/tests/unit/schedule/replications.js
A       sentinel/tests/unit/sendable.js
Falling back to patching base and 3-way merge...
CONFLICT (rename/delete): demo-forms/sentinel/tests/unit/sendable.js deleted in         85804e8d6fad7133684aabd46ad0a9566b11b23e and renamed in Migrate all medic-docs r        eferences #6489. Version Migrate all medic-docs references #6489 of demo-forms/s        entinel/tests/unit/sendable.js left in tree.
Auto-merging demo-forms/sentinel/server.js
Auto-merging .github/PULL_REQUEST_TEMPLATE.md
CONFLICT (content): Merge conflict in .github/PULL_REQUEST_TEMPLATE.md
error: Failed to merge in the changes.
Patch failed at 0001 Migrate all medic-docs references #6489
The copy of the patch that failed is found in: .git/rebase-apply/patch

When you have resolved this problem, run "git rebase --continue".
If you prefer to skip this patch, run "git rebase --skip" instead.
To check out the original branch and stop rebasing, run "git rebase --abort".

**Comment by latin-panda** [comment]
Hi @ronna 

Thank you for your contribution, it's really helpful and nice to see more people joining the community! ✨✨

Since the goal is that this PR only contains changes related with the ticket, I was thinking that there are a couple of ways we can try to achieve this by resolving the conflicts:

1.. Reverting the unwanted changes manually:
For example in your branch: If there's a change indicating a rename of file and it isn't related with the ticket, then you can try to manually put back the original name or move back the file to the original place. Commit and push the work and check again if this change is gone from the PR. I would suggest to try first with one file and see if that works before continuing with the same approach. 

If we have tried everything and we're still unsuccessful to resolve these conflicts, the other option is:
2..  Create a new branch from the latest code in master and reapply your changes. 
This requires a bit of rework but it's safe to do. However, I suggest to work together on figuring out how to resolve this by using the current branch 👍

Lastly, I would like to share this video I like about [merge conflicts in git](https://www.youtube.com/watch?v=JtIX3HJKwfo) any text editor will work but there are some that provides tools that helps visualising merge conflicts easier. VS Code has some plugins, Webstorm has a nice tooling for this too.

Looking forward to hearing your thoughts on this. Happy coding!

**Comment by ronna** [comment]
Hi @latin-panda & @abbyad 

please find new pull request for your review #6682

**Pull request closed without merging**